File: LeetCode/1431. Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        
        
        int s=candies.size();
        vector<bool> v(s);
        for(int i=0;i<s;i++)
        {
            if(candies[i]+extraCandies> *max_element(candies.begin(), candies.end())-1)
                v[i]=true;
            else
                v[i]=false;
        }
        return v;
    }
};

File: LeetCode/1431-Kids-With-the-Greatest-Number-of-Candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int n=candies.length;

        int maxVal=0;
        for(int i=0; i<n; i++){
            maxVal = Math.max(candies[i], maxVal);
        }

        List<Boolean> resp = new ArrayList<>();
        for(int i=0; i<n; i++){
            resp.add((candies[i] + extraCandies)>=maxVal);
        }
        return resp;
    }
}